test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -45,67 +45,62 @@ class configuration;
 
 namespace test_harness {
 
-class runtime_statistic {
+class statistics {
     public:
-    explicit runtime_statistic(configuration *config);
-    virtual ~runtime_statistic() = default;
+    statistics() = default;
+    explicit statistics(configuration *config, const std::string &stat_name, int stat_field);
```

Comment:
Maybe we should make `configuration` a reference, not a pointer? Otherwise, we need to specify that this class doesn't store the pointer.

Reply:
I agree, we are expecting a reference from aa3cd46e1c8710696e342810c848060498ad6511.